docs/general/dns-filtering-syntax.md

```diff
@@ -202,7 +202,7 @@ The problem with this approach is that this way you will also unblock tracking d
 
 - `@@*$denyallow=com|net`: unblock everything except for `*.com` and `*.net`.
 
-- `||example.org^$denyallow=sub.example.org`. block `example.org` and `*.example.org` but don't block `sub.example.org`.
+- `||example.org^$denyallow=sub.example.org`. block `example.org` and `*.example.org` but doesn't block `sub.example.org`.
```

Comment:
```suggestion
- `example.org^$denyallow=sub.example.org`: block `example.org` and `*.example.org`, but not `sub.example.org`.

Reply:
I guess it's simple infinitive, so the verb doesn't need "s". otherwise, you'd need to add it to "block" too